styles/cart.css

```diff
@@ -62,7 +62,7 @@
     box-shadow: rgba(0, 0, 0, 0.35) 0px 7px 14px;
 }
 
-.coupons-btn, .checkout-btn{
+.coupons-btn, .checkout-btn, .seemore-btn{
     background-color: var(--primary-clr);
     border:none;
     border-radius: 0.5rem;
```

Comment:
I don't get the point of giving same styling to different classes name a single class for its feature and use it on different places. Like these all classes are having same styles name them as one generic class use it at multiple places

Reply:
Have refactored it to a generic class and made the changes.